Lib/dataclasses.py

```diff
@@ -239,6 +241,9 @@ def __repr__(self):
     def __class_getitem__(cls, type):
         return InitVar(type)
 
+    def __call__(self, *args, **kwds):
+        raise TypeError(f"Cannot instantiate {self!r}")
```

Comment:
Same here

Reply:
This needs a comment to explain why it's here.

Reply:
I added a comment explaining the origin of why this is needed.